ghost/core/core/server/web/shared/middleware/api/spam-prevention.js

```diff
@@ -123,6 +126,32 @@ const globalReset = () => {
     return globalResetInstance;
 };
 
+const webMentionsBlock = () => {
```

Comment:
Minor but "webmention" should be cased as `webmention` rather than `webMention` because it's one word